scripts/tests/rosetta-load.sh

```diff
@@ -485,7 +485,7 @@ function assert_memory_usage_within_thresholds() {
     
     # Check PostgreSQL memory usage
     local postgres_memory
-    postgres_memory=$(ps -p $(pgrep -d, -f postgres) -o rss= 2>/dev/null | awk '{sum+=$1} END {print sum/1024}' 2>/dev/null)
+    postgres_memory=$(ps -u postgres -o rss= 2>/dev/null | awk '{sum+=$1} END {print sum/1024}' 2>/dev/null)
```

Comment:
I suggest remove `2>/dev/null` here because if this command runs correctly it shouldn't have stderr output.